key.core/src/main/resources/de/uka/ilkd/key/smt/solvertypes/Z3_QF.props

```diff
@@ -0,0 +1,22 @@
+params=-in -smt2
+timeout=-1
+name=Z3_QF
+info=Z3 solver by Microsoft. Translation excludes quantifiers, making this variant much faster for certain problems.
```

Comment:
Since axioms from the preamble can still contain quantifiers (only sequent formulas with quantifiers do not get translated), we agreed that the description should reflect this.